addon/components/custom-submission-form-fields/bestuursorgaan-selector/show.js

```diff
@@ -23,10 +24,19 @@ export default class CustomSubmissionFormFieldsBestuursorgaanSelectorShowCompone
     });
   }
 
+  getOptionPredicates() {
+    return {
+      conceptScheme: EXT('conceptScheme'),
+    };
+  }
+
   loadOptions() {
     const metaGraph = this.args.graphs.metaGraph;
-    const fieldOptions = this.args.field.options;
-    const conceptScheme = new namedNode(fieldOptions.conceptScheme);
+    const conceptScheme = this.findFieldOption('conceptScheme', 'node');
+
+    if (!conceptScheme) {
+      return;
+    }
```

Comment:
Before there was no check if the conceptscheme was not null. This is now added. ok?

Reply:
I think so, yes. What did it do now if the conceptScheme wasn't set? The .match would see `undefined` as a wildcard, right? So potentially it would return concepts from other conceptSchemes as well?

If so this is just a bugfix I think?

Reply:
Yes indeed it would use undefined in the match what results in getting all values that have the subject and inscheme predicate.

Bug fix indeed 🆗